packages/nimble-components/src/accordion/specs/IxD.md

```diff
@@ -0,0 +1,193 @@
+# Nimble Accordion (IxD)
+
+## Overview
+
+The accordion component is used to present a large amount of information in a small space via progressive disclosure. Accordions make information easier to parse when not all of the information may be relevant or important to the user at the same time.
+n
+
+## Background
+
+-   [IxD recommendation for Tree, Accordion, Disclosure, Table row details (#1150)](https://github.com/orgs/ni/projects/7/views/13?pane=issue&itemId=24667320)
+-   [ViD Accordion Styles (Figma)](https://www.figma.com/file/PO9mFOu5BCl8aJvFchEeuN/Nimble_Components?type=design&node-id=1295-85131&mode=design)
+
+## Usage
+
+### Accordions & Disclosures
```

Comment:
I have an accordion component and a disclosure component because the ARIA patterns include both. Fast controls do as well. Disclosures can have lots of different permutations than just looking like an expander panel. I guess functionally, the accordion doesn't necessarily have to have literal disclosure components inside of it.